src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -1257,6 +1263,15 @@ private fun RsCallExpr.expectedParamsCount(): Pair<Int, Boolean>? {
             val s = if (el.selfParameter != null) 1 else 0
             Pair(count + s, el.isVariadic)
         }
+        is RsPatBinding -> {
```

Comment:
Note, I can write
```rust
struct S(&'static dyn Fn(i32));

const F: &'static dyn Fn(i32) = &bar;

fn main() {
    let s = S(&bar);
    s.0(123, 123); // <- E0057 and struct field
    F(1, 2); // <- E0057 and const item
}

fn bar(x: i32) {
    println!("{}", x);
}
```

Just notice, don't insist on supporting these cases in this PR